public/js/publication-api-buttons.js

```diff
@@ -100,6 +107,65 @@ function renderFetchEntitlementsPlansButton(selector) {
   document.querySelector(selector).appendChild(button);
 }
 
+
+/**
+ * renderFetchMemberButton
+ * Creates a button to fetch entitlements manually
+ * @param {string} selector
+ */
+function renderFetchMemberButton(selector) {
+  const accessToken = getAuthzCred('accessToken');
+  const button = document.createElement('button');
+  button.classList.add('btn', 'btn-primary');
+  button.onclick = async () => {
+    const loaderOutput = document.createElement('div');
+    document.querySelector('#GISOutput').append(loaderOutput);
+    const loader = new Loader(loaderOutput);
+    loader.start();
+    const entitlements = await queryLocalEntitlements(accessToken);
+    const readerId = entitlements.entitlements[0].readerId;
+    const readerData = await queryMemberData(accessToken, readerId);
+    loader.stop();
+    insertHighlightedJson(
+        '#GISOutput', readerData, 'Member data for the current user');
+  };
+  button.innerText = 'Query member data';
+  document.querySelector(selector).appendChild(button);
+}
+
+
+/**
+ * renderFetchOrderButton
+ * Creates a button to fetch order details manually
+ * @param {string} selector
+ */
+function renderFetchOrderButton(selector) {
+  const accessToken = getAuthzCred('accessToken');
+  const button = document.createElement('button');
+  button.classList.add('btn', 'btn-primary');
+  button.onclick = async () => {
+    const loaderOutput = document.createElement('div');
+    document.querySelector('#GISOutput').append(loaderOutput);
+    const loader = new Loader(loaderOutput);
+    loader.start();
+    const entitlements = await queryLocalEntitlements(accessToken);
```

Comment:
Maybe at some point we could allow for an input for `readerId` instead of pulling from GIS given this is the way that publishers will use this on the server?

Reply:
Agreed, and this will make sense after I move to the service account auth method.